packages/integration-tests/src/tests/ui/smoke.test.ts

```diff
@@ -1,8 +1,38 @@
-import { logtoUrl } from '#src/constants.js';
+import { logtoUrl, consoleUsername, consolePassword } from '#src/constants.js';
 
 describe('smoke testing', () => {
-  it('opens with app element', async () => {
-    await page.goto(new URL('/sign-in', logtoUrl).href);
+  it('opens with app element and navigates to sign-in page', async () => {
+    await page.goto(new URL('console', logtoUrl).href);
+    await page.waitForNavigation({ waitUntil: 'networkidle2' });
+
     await expect(page.$('#app')).resolves.not.toBeNull();
+    expect(page.url()).toBe(new URL('sign-in', logtoUrl).href);
+  });
+
+  it('signs in to admin console', async () => {
```

Comment:
be better to have each `it` separated?

Reply:
As discussed offline, we now assume the test cases in the same `describe` are sharing the same context. No need to separate `it` for the time being